src/components/FolderCard.jsx

```diff
@@ -85,10 +104,10 @@ const FolderCard = ({
       }
       <Link className={`${contentStyles.component} ${contentStyles.card} ${elementStyles.folderCard}`} to={generateLink()}>
         <div id={itemIndex} className={`${contentStyles.folderInfo}`}>
-          <i className={`bx bx-md text-dark ${isHomepage ? 'bxs-home-circle' : 'bxs-folder'} ${contentStyles.componentIcon}`} />
+          <i className={`bx bx-md text-dark ${generateImage(pageType)} ${contentStyles.componentIcon}`} />
           <span className={`${contentStyles.componentFolderName} align-self-center ml-4 mr-auto`}>{displayText}</span>
           {
-            isHomepage
+            pageType === 'homepage'
```

Comment:
Should also check for 'contact-us', since that page shouldn't have rename/delete folder options as well